Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "iomrascálaí"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
And apart from which version to use, I'd like to have a separate pull request for this. We also need to change the version in `version.rs`, add a tag, etc.